### PR TITLE
add dispatcher model binding doc

### PR DESCRIPTION
**Pull request by mattvb91**
The associated api update for BindModelInterface is included in https://github.com/phalcon/docs/pull/784

### CODE REVIEW

**Comment by SidRoberts** [comment]
Thanks, I'll merge it once 2.1 is released.

**Comment by sergeyklay** [comment]
@SidRoberts

File: en/reference/dispatching.rst [hunk 1]
```diff
@@ -456,6 +456,83 @@ before dispatch the action preparing the parameter accordingly:
 The above example has been simplified for academic purposes.
 A developer can improve it to inject any kind of dependency or model in actions before be executed.
 
+From 2.1.x onwards the dispatcher also comes with an option to handle this internally for all models passed into a controller action.
```
Comment (Green-Cat) [inline en/reference/dispatching.rst]: This should probably say "3.0 onwards" as 2.1.x never shipped.

Comment (sergeyklay) [inline en/reference/dispatching.rst]: Since Phalcon 2.1.0 beta 1 :)

**Pull request merged**